src/services/app/user.ts

```diff
@@ -496,6 +500,7 @@ export default class UserService {
         return {
             isBeneficiary: beneficiary !== null,
             isManager: manager !== null || managerInPendingCommunity,
+            readBeneficiaryRules,
```

Comment:
I was looking at this, thinking about the next steps and I just wonder if we could do this differently.
There are some results here to be deprecated (I've just added tasks for that). And we are returning more beneficiary information. I was thinking about changing the result. What do you think of this

```
return {
    beneficiary,
    manager,
    user
}
```
At this point, `beneficiary` would only contain `blocked`, `readRules` and `communityId`. `manager` would contain `communityId`only. And `user` would contain `suspect` only.
This requires some changes here and some changes in `beneficiary` and `manager` table (to which I added tasks on JIRA) but still, looks more organized and simpler to maintain. Any thoughts?
In my opinion, we could start with this approach right away (now) and one UI side we will only deal with beneficiary.readRules for now. Then we add another task and later we do this change.

Reply:
I think that is a good structure to return, more clear. 
I added a comment in all the other attributes to be deprecated